src/adapters/AsyncContentEditableAdapter.ts

```diff
@@ -64,12 +64,12 @@ export class AsyncContentEditableAdapter extends AbstractRichtextEditorAdapter i
     assertElementIsDisplayed(this.getEditorElement());
     const [alignedMatches] = this.selectMatches(checkId, matchesWithReplacement);
     const replacement = alignedMatches.map((m) => m.originalMatch.replacement).join('');
+    simulateInputEvent(this.getEditorElement(), 'beforeinput');
```

Comment:
Added config check this.config.disableInputEventSimulation

Reply:
Fixed